transcriptclean.wdl

```diff
@@ -0,0 +1,264 @@
+version 1.0
+
+# Copyright (c) 2019 Sequencing Analysis Support Core - Leiden University Medical Center
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+
+task CleanSpliceJns {
+    input {
+        File SAMfile
+        File referenceGenome
+        String outputPrefix
+        String outputDirPath
+        File spliceJunctionAnnotation
+
+        File? variantFile
+
+        Int cores = 1
+        Int memory = 4
+        String dockerImage = "biocontainers/transcriptclean:v1.0.7_cv1"
+    }
+
+    command {
+        set -e pipefail
+        mkdir -p ~{outputDirPath}
+        clean_splice_jns \
+        ~{"--f=" + SAMfile} \
+        ~{"--g=" + referenceGenome} \
+        ~{"--o=" + outputDirPath + outputPrefix} \
```

Comment:
Minor detail, but outputDirPath has to have a "/" defined in the input. This might cause unexpected behavior if someone were to define the ouputDir like `/some/path` instead of `/some/path/`